content/es/privacy.md

```diff
@@ -21,13 +21,13 @@ En esta política de privacidad, nos gustaría explicarte qué datos recopilamos
 
 Esta política de privacidad se aplica a todas las actividades de Datenanfragen.de e.&nbsp;V. (“la asociación").
 
-Esto incluye los datos que recopilamos y procesamos de nuestros miembro, pero también los datos que se obtienen de donaciones y similares.
+Esto incluye los datos que recopilamos y procesamos de nuestros miembros, pero también los datos que se obtienen de donaciones y similares.
 
-Además, esto incluye nuestros sitios web [solicituddedatos.es](https://www.solicituddedatos.es), [Datenanfragen.de](https://www.datenanfragen.de), [datarequests.org](https://www.datarequests.org), [demandetesdonnees.fr](https://www.demandetesdonnees.fr) y [pedidodedados.org](https://www.pedidodedados.org/).
+Además, esto incluye nuestros sitios web [solicituddedatos.es](https://www.solicituddedatos.es), [datenanfragen.de](https://www.datenanfragen.de), [datarequests.org](https://www.datarequests.org), [demandetesdonnees.fr](https://www.demandetesdonnees.fr) y [pedidodedados.org](https://www.pedidodedados.org/).
```

Comment:
We think of "Datenanfragen.de" as a proper name that should always be capitalized, so I'll change this back.

```suggestion
Además, esto incluye nuestros sitios web [solicituddedatos.es](https://www.solicituddedatos.es), [Datenanfragen.de](https://www.datenanfragen.de), [datarequests.org](https://www.datarequests.org), [demandetesdonnees.fr](https://www.demandetesdonnees.fr) y [pedidodedados.org](https://www.pedidodedados.org/).
```